osu.Game/Localisation/SongSelectStrings.cs

```diff
@@ -19,6 +19,11 @@ public static class SongSelectStrings
         /// </summary>
         public static LocalisableString LocallyModifiedTooltip => new TranslatableString(getKey(@"locally_modified_tooltip"), @"Has been locally modified");
 
+        /// <summary>
+        /// "{0} beatmaps displayed"
+        /// </summary>
+        public static LocalisableString BeatmapsDisplayed(int arg0) => new TranslatableString(getKey(@"beatmaps_displayed"), @"{0:#,0} beatmaps displayed", arg0);
```

Comment:
This is the first instance I can find of having string format specs inside localisable strings. Discussing with @smoogipoo this seems okay (allows translators to adjust as required for each language) but worth nothing and scrutinising.

Reply:
The format specs bother me less than the fact that this string is not correctly translatable anyway, because it has a numeral in it and therefore requires being able to specify plural forms for grammatically correct translation. As in, if we want to have this localisable, we probably need to resurrect or rewrite https://github.com/ppy/osu-framework/pull/4918 first.

In fact as is, this is wonky even in English ("1 beatmaps displayed").

Reply:
I think not localising for now is best. Most of song select is not localised anyway.